docs/shader-modules/project.md

```diff
@@ -18,7 +18,7 @@ attribute float instanceSize;
 
 void main(void) {
   vec3 center = project_position(instanceCenter);
-  vec3 vertex = positions * project_scale(instanceSize);
+  vec3 vertex = positions * project_size(instanceSize);
```

Comment:
+1 for renaming this.

I always intended the naming of the functions here to align with the concepts introduced in the coordinate system article in deck.gl. The idea being that these functions would then be defined to implement what is described on the user level.

- that article currently doesn't talk about sizes, it talks about "positions" and "distances", so either add a mention of sizes there or change to "distance" here.

Longer term, there is also a model matrix implication on relative distances (rotating hexagons etc). Even if out of scope for this PR it would be good to have a sense/path forward for how to handle that. Perhaps we need both concepts (project_distance and project_size)?